src/components/ec-inline-input-field/ec-inline-input-field.vue

```diff
@@ -71,6 +83,11 @@ export default {
       valueForLoading: this.value,
     };
   },
+  computed: {
+    isCopyingAllowed() {
```

Comment:
not sure about this naming re: consistency (we don't call `isEditionAllowed` to `isEditable`)

Reply:
I see your point. We have the prop "isCopiable" and I wanted to differentiate. I used "allowed" because even though can be copiable doesn't mean is allowed if is also editable.